### DISCUSSION TITLE
Set a background image to all users

### DISCUSSION THREAD

**Discussion by YarinShitrit** [open-webui/open-webui#7412] [Q&A]
Hey,

Is it possible to set the "Chat Background Image" in a way that will affect all users?

As for now I found that it is only possible for each user to change the background image by going to settings -> interface - > Chat Background Image

Thanks

**Comment by apunkt** [comment]
+1

**Comment by dsewell** [comment]
I have raised a PR for this change - https://github.com/open-webui/open-webui/pull/10073

**Comment by OdinVex** [comment]
The Pull Request was closed instead of accepted. Direct quote: "I'll be adding this feature personally and this feature will be introduced as a premium feature, Feel free to maintain a forked version!"

**Comment by nboehlke** [comment]
+1

**Comment by LarryStewart2022** [comment]
Has this ever been done?

**Reply by nboehlke** [reply]
Yes, but its an enterprise plan feature (as far as i know)
